### PR TITLE
[fix] Display the date in a correct format

### PR DESCRIPTION
**Pull request by zamentur**
### Problem

The date is send in python format, not good for a Web API
https://cloud.githubusercontent.com/assets/41827/16403714/5501e2a4-3cf8-11e6-8c2e-a77d57ab5a7f.png

### Solution

Display the date correctly, in a usable format

### How to test
This PR use date: https://github.com/YunoHost/yunohost/pull/165 
The webadmin PR associated is : https://github.com/YunoHost/yunohost-admin/pull/130

## Validation

- [ ] Principle agreement 0/2 : 
- [ ] Quick review 0/1 : 
- [ ] Simple test 0/1 : 
- [ ] Deep review 0/1 :

### CODE REVIEW

**Comment by Psycojoker** [comment]
Shouldn't we return a unix timestamp instead to be able to parse it in javascript and play with it in js?

Or mutliple format? Like one wchich is `str(o)` and one which is a timestamp?

File: moulinette/utils/serialize.py [hunk 1]
```diff
@@ -24,6 +25,10 @@ def default(self, o):
                 hasattr(o, '__iter__') and hasattr(o, 'next')):
             return list(o)
 
+        # Convert compatible containers into list
+        if isinstance(o, datetime.datetime) or isinstance(o, datetime.date):
+            return str(o)
```
[the remark this answers is not in the document]
Comment (Psycojoker) [reply moulinette/utils/serialize.py]: Is it easily parsable in javacsript if we wants to operate on dates?

[the remark this answers is not in the document]
Comment (randomstuff) [reply moulinette/utils/serialize.py]: In practice yes. `new Date("2018-01-01")` gives `2018-01-01T00:00:00.000Z` and `new Date("2018-01-01T14:10:00+02:00")` gives `2018-01-01T12:10:00.000Z`.

[In theory](https://momentjs.com/docs/#/parsing/string/), the formats recognized by `Date` is not properly specified. In practice, ISO-8601 is recognized AFAIU.

[the remark this answers is not in the document]
Comment (randomstuff) [reply moulinette/utils/serialize.py]: By the way this is consistent with `(new Date("2018-01-01T14:10:00+02:00")).toJSON()` which gives `2018-01-01T12:10:00.000Z`.

[the remark this answers is not in the document]
Comment (Psycojoker) [reply moulinette/utils/serialize.py]: Great, that looks like what we want :)

[the remark this answers is not in the document]
Comment (Psycojoker) [reply moulinette/utils/serialize.py]: Thx!

[the remark this answers is not in the document]
Comment (zamentur) [reply moulinette/utils/serialize.py]: I agree

**Pull request merged**